### DISCUSSION TITLE
rigtools not working with goguadrian

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1244] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When I open rigtools and put in goguadrian as the extension I'd it opens the link which Is blocked by goguadrain so I started goguadrain from loading in and that worked for a while until it loads in and get blocked too fast for  me to stop it , could someone help

**Comment by no-name-6655** [comment, marked as answer]
If it's blocked by Goguardian then the exploit will not work.

**Reply by ghost** [reply]
but before I could hit the refresh  button and stop it from loading in and could use rigtools but now after I stop it it still reloads and get blocked

**Reply by no-name-6655** [reply]
I don’t understand what do you mean by refreshing goguardian? Like stopping it in task manager?

**Reply by ghost** [reply]
once the page load in a hit the x to stop it from loading you know the refresh button is

**Reply by no-name-6655** [reply]
From what I understand I think your admin put a manual block on the link that makes rigtools work, using goguardian.  If your admin put a block on it, then there's no way around that without temporarily stopping goguardian so that you can remove it using rigtools. If I am wrong about this then can you send an image of the problem because im kind of confused.

**Reply by ghost** [reply]
I'm send a video

**Reply by ghost** [reply]
[Screen recording 2024-11-06 3.06.08 PM.webm](https://github.com/user-attachments/assets/52696c11-5ea0-4717-aaa6-4d92b46cc722)


see when I hit boss and I im hit the refresh button to stop goguardian from blocking it , but  then goguardian just loads in again , it used to never do that

**Reply by no-name-6655** [reply]
At 0:07 can't you just click 'Disable injected extension'? It looks like you have about 1 second to click that button before it gets blocked, it should be enough time if you are quick.

**Reply by ghost** [reply]
k thanks

**Reply by must9f** [reply]
Try to hang the extension to freeze it with exthanger, everything will be slow but you can freeze go guardian for like 5 mins

**Comment by S-PScripts** [comment]
i've heard of goguardian but not goguadrain!/j

**Reply by ghost** [reply]
I suck at typing don't judge

**Comment by ghost** [comment]
also could the dev link or what the rigtools link method me used the to run code on any website bc when you open rigtools from the link under the network thing and enter the extension I'd it runs code on that extension manifest file and then redirects to the extension rigtools page how can it do that ?? could make it run on any website and instead of it redirecting us to the extensions version of rigtools
